docker/base/Dockerfile

```diff
@@ -1 +1,7 @@
 FROM redis:6.0.6-buster
+
+RUN apt update && apt install openssl
+
+COPY gen-test-certs.sh /tests/gen-test-certs.sh
```

Comment:
This is a bit of a PITA. Takes a while to generate these test certs when you have to build the base image. But this way they won't expire.